src/EnergyPlus/CommandLineInterface.cc

```diff
@@ -469,7 +469,7 @@ namespace CommandLineInterface {
         state.files.rdd.filePath = composePath(normalSuffix + ".rdd");
         state.dataStrGlobals->outputShdFilePath = composePath(normalSuffix + ".shd");
         state.files.dfs.filePath = composePath(normalSuffix + ".dfs");
-        state.dataStrGlobals->outputGLHEFilePath = composePath(normalSuffix + ".glhe");
+        state.dataStrGlobals->outputGLHEFilePath = composePath(normalSuffix + "_glhe.cbor");
```

Comment:
Hmm, the regression tool does check the `.glhe` file if it exists.  If this change is for every simulation, then the regression tool will no longer do any results comparison on the glhe cache output.  Is there a reason it is being converted to cbor here?  Or is this only in a certain part of the code and I'm just missing the context?

Reply:
I didn't realize the regression tool checks the `.glhe` file. I thought it was just an internal EP cache file and thus a low risk/impact change. I wanted to get rid of the custom `.glhe` file extension and just use `.json` or `.cbor`, which is why I made the change. Then since it is a cache file, I thought moving to `cbor` would make sense to read/write the file faster. Anyways, I can revert this for now and create a new PR to convert from `.glhe` after release.